plugins/analytics-module-ga/README.md

```diff
@@ -0,0 +1,131 @@
+# Analytics Module: Google Analytics
+
+This plugin provides an opinionated implementation of the Backstage Analytics
+API for Google Analytics. Once installed and configured, analytics events will
+be sent to GA as your users navigate and use your Backstage instance.
+
+This plugin contains no (and will never contain any) other functionality.
+
+## Installation
+
+1. Install this plugin in your Backstage instance: `yarn add @backstage/plugin-analytics-module-ga`
+2. Register the plugin with your App. In most instances of Backstage, this is
+   as simple as adding `export { analyticsProviderGA } from '@backstage/plugin-analytics-module-ga';`
+   to your `packages/app/src/plugins.ts` file.
+3. Configure the plugin (see below).
+
+## Configuration
+
+The following is the minimum configuration required to start sending analytics
+events to GA. All that's needed is your Universal Analytics tracking ID:
+
+```yaml
+# app-config.yaml
+app:
+  analytics:
+    provider: ga
```

Comment:
What is the intent behind this `provider` config property? When one adds the `@backstage/plugin-analytics-module-ga` dependency and setups the plugin export, it is always used regardless of this property.

Reply:
I suppose I was thinking to follow a similar pattern in other Backstage plugins where there's a key that allows one to differentiate config schemas under a common key.  Like...

```typescript
export interface Config = {
  app: {
    analytics?:
      // Google Analytics
      | {
        provider: 'ga',
        ga: {
          trackingId: string;
        }
      // Amplitude
      | {
        provider: 'amplitude';
        amplitude: {
          apiKey: string;
        };
      };
  };
};
```

But maybe this isn't necessary or valuable if the configs are spread across different plugins.

Reply:
Ended up removing this.